### DISCUSSION TITLE
Group icons on the map

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#147] [Q&A] [answered]
This feature already exists on Android Cruiser in "Settings | Map | Advanced | Marker groups (zoom)".

---

Next desktop Cruiser will have the marker clusters enabled
to avoid slow rendering with many favorites in small zooms.

![Cruiser](https://user-images.githubusercontent.com/3484020/235350212-5f0bd3cf-4262-4553-8388-36a9e41e6aa1.jpg)

**Comment by devemux86** [comment]
A new feature that marker groups will have in the next update of Cruiser (Android + Desktop)
is that when tap on a group (e.g. 200+), the map will zoom to show the markers in the group.

New favorite categories can fill the map with markers and performance is always important! :slightly_smiling_face: 
- https://github.com/devemux86/cruiser/discussions/45#discussioncomment-5791140

**Reply by devemux86** [reply]
> when tap on a group (e.g. 200+), the map will zoom to show the markers in the group

It is available in Cruiser [3.0.20](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Comment by devemux86** [comment, marked as answer]
> when tap on a group (e.g. 200+), the map will zoom to show the markers in the group.

Implemented in [Cruiser 3.0.19](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by DolomitesTom** [reply]
works

**Comment by devemux86** [comment]
Please enable and use the "Marker groups", especially **if you display too many favorites on the map**.

**Reply by DolomitesTom** [reply]
yes setting is on zoom 11

**Comment by DolomitesTom** [comment]
Export of favorite groups only works group by group. Is it possible to export and import all with managed groups?

**Reply by devemux86** [reply]
Please see here:

- https://github.com/devemux86/cruiser/discussions/45#discussioncomment-5875901

**Reply by DolomitesTom** [reply]
OK thanks, understand, not used that often, therefore no priority. Maybe for somewhen you don't know anymore what to develop 😉.
